.github/workflows/percy-prepare.yml

```diff
@@ -27,24 +27,29 @@ jobs:
             templates/docs/examples/
             templates/_macros/
             scss/
+            tokens/
+            sd.config.json
           ref: ${{ github.event.inputs.commitsh }}
           repository: ${{ github.event.inputs.repository }}
 
       - name: Populate artifact directory
         run: |
           # Create needed directories so `cp` doesn't fail
+          # If a PR is made that doesn't have these directories in the source, the CPs at the end of the step will fail
           mkdir -p artifact
+          mkdir -p tokens
           mkdir -p templates/docs/examples/
           mkdir -p templates/_macros/
           mkdir -p templates/macros/
+          touch sd.config.json
```

Comment:
Is the touch needed?

Reply:
`touch` shouldn't be needed as it's not moved by the percy snapshot workflow and tokens build errors are ignored following this PR, and this PR introduces `sd.config.json` to the repo so any PR that has this commit in its history should not fail.

I'll try it out without the `touch`

Reply:
Works fine without `touch`!